src/plugins/ovp/components/video-screen/Medialive.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import useNode from '@triniti/cms/plugins/ncr/components/useNode.js';
+
+const Medialive = ({ nodeRef }) => {
+
+  const { node } = useNode(nodeRef, false);
+
+  if (!node || !node.schema().hasMixin('triniti:ovp:mixin:transcodeable')) {
+    return '';
+  }
+
+  const status = node && node.has('transcoding_status') ? node.get('transcoding_status').getValue() : 'unknown';
```

Comment:
@rsumilang feels a bit odd to just render a title modification to another card.  i think all the media live stuff should in this card and should not be mixed in with the other details/cards.

Reply:
@gdbrown ok i did a component thing. let me know if that looks good